vulpes/memory/gamestate/table.hpp

```diff
@@ -8,6 +8,8 @@
 
 #include <vulpes/memory/types.hpp>
 
+#pragma pack(push, 1)
```

Comment:
What's the rationale behind enabling this? [According to this thread](https://stackoverflow.com/questions/3318410/pragma-pack-effect), this setting can potentially cause problems with accessing the memory properly.

Reply:
The rationale is that compiler repacking can't screw our structs. It has happened before. Most compilers tend to pack by the size of the biggest value in the table. Like if I add a double then everything that was supposed to be 4 aligned is now screwed up, our size assertions fail. We all burn in hell.

This also makes the structures easier to align with the actual memory. Because packing really is just weird if not 1:1 like this.

Reply:
We don't ever do this for our own structs though. Only for structs that we read from the game.

Reply:
What you typed there should be put in a comment for future maintainers